### DISCUSSION TITLE
POD discussion: Eulerian storm tracker

### DISCUSSION THREAD

**Discussion by tsjackson-noaa** [NOAA-GFDL/MDTF-diagnostics#124] [Q&A]
Issue thread to document progress on incorporating/maintaining the Eulerian storm tracker POD; cc @jeyavinoth and @jfbooth. Associated pull request is #50.

**Comment by tsjackson-noaa** [comment]
Hi @jeyavinoth, so far the model plots look good!

- Please point me to where I can download the obs data (better to email me than to make this info public) so I can verify the other plots.
- [plotter.plot](https://github.com/NOAA-GFDL/MDTF-diagnostics/blob/d093d0be35cd272937de3ec66135ab8409d2bba3/diagnostics/eulerian_storm_track/plotter.py#L60)'s contour plots have a seam at lon=0. You may wish to consider using [cartopy.util.add_cyclic_point](https://stackoverflow.com/questions/50089485/seam-where-longitude-wraps-around-in-matplotlib-cartopy-mollweide-projection) to fix this.

**Comment by jeyavinoth** [comment]
Thanks @tsjackson-noaa 

- I sent you an email with a dropbox link to the obs_data used to create the plots. 
- I got rid of the seam at lon=0 using your suggestion. I pushed the changes to my fork.

**Comment by tsjackson-noaa** [comment]
@jeyavinoth - thanks for the updates. Everything looks great!

**Comment by jkrasting** [comment]
Glad the initial tests look good @tsjackson-noaa and @jeyavinoth - we still need to a do a test inside the firewall on GFDL data though.

**Comment by jkrasting** [comment]
@tsjackson-noaa - any updates on the internal testing?

**Comment by jeyavinoth** [comment]
@jkrasting @tsjackson-noaa Do you have any further updates on the internal testing?

**Comment by tsjackson-noaa** [comment]
The branch containing existing development work on this POD has been renamed from pod/eulerian-storm-track to [feature/add_eulerian_storm_track](https://github.com/NOAA-GFDL/MDTF-diagnostics/tree/feature/add_eulerian_storm_track), as part of the reorganization proposed in issue NOAA-GFDL/MDTF-diagnostics#106.

**Reply by jfbooth** [reply]
@tsjackson-noaa regarding feature/add_eulerian_storm_track. This is a POD that we submitted and had approved in Jan 2021. But then, after V3 came out, the POD did not get moved into the main MDTF set of PODs. Is there more work that you want us to do on this POD before it is included in the main package? (Sorry to bring up something so old, we got busy working on the etc_composite POD and forgot to keep track of this other POD that we had submitted.) Thanks!

**Reply by wrongkindofdoctor** [reply]
@jfbooth Tom is no longer at GFDL, so you can ping @wrongkindofdoctor or @jkrasting in any messages moving forward.

The feature branch Tom created contained the add_temp_extremes_distshape POD, which has been merged into the repo and added to the GitHub Actions CI (see PR #273). It does not contain any code for an eulerian_storm_track POD, and there is no branch for this in Tom's repo. I was not aware of that you had submitted the POD, and can't find a PR for it (the closed draft PR for the feature/add_eularian_storm_track  branch does not contain the code). I'm going to take a guess that Tom was going to add it to the branch, but did not get to it before he left. 

I see that @jeyavinoth has a branch for the diagnostic in his fork: https://github.com/jeyavinoth/MDTF-diagnostics/tree/feature/eulerian-storm-track/diagnostics. If he can merge in updates from the NOAA_GFDL/develop branch to this branch, resolve any conflicts if necessary, and submit a PR to the NOAA_GFDL/develop branch, John and I can review it and get it in.
